charts/kminion/templates/deployment.yaml

```diff
@@ -24,7 +27,7 @@ spec:
       {{- end}}
       labels:
         {{- include "kminion.selectorLabels" . | nindent 8}}
-        {{- if .Values.customLabels}}
+        {{- if .Values.customLabels -}}
```

Comment:
I think that's not needed

Reply:
this removes an extra empty line when the chart is rendered

Reply:
but happy to remove it

Reply:
I have remove it